Add conda-forge to installation instructions (#228)
pyRiemann/pyRiemann: README.md

pip install git+https://github.com/pyRiemann/pyRiemann
```

#### Using conda

The package is distributed via [conda-forge](https://conda-forge.org).

> We recommend installing it into a new, dedicated environment.

Reviewer: why a new env? then you end up with one env per package and it makes no sense.

I would say "to activate the environment" you are willing to update if it's not your base environment and then just conda install

my 2c

> We recommend installing it into a new, dedicated environment.
> The following command will create an environment named `pyriemann`.

Reviewer 2: ```suggestion
You could install it in your working environment, with the following command:
```

I agree, pyriemann do not need to install in a specific env.
